### DISCUSSION TITLE
Add a sample in SampleCompose showcasing how to use AppNavigation for Deep Linking

### DISCUSSION THREAD

**Discussion by kaungkhantsoe** [nimblehq/android-templates#442] [RFCs]
## Issue

Current SampleCompose contains samples to navigate to a screen by passing an `id` or a `parcelable object`. We still don't have an example of how to navigate using DeepLink. Shall we add a sample for it?

## Solution

- Create a notification channel
- Show notification with a deep link to the SecondScreen
- Showcase clicking on the notification will redirect to SecondScreen

## Who Benefits?

Android Developer

**Comment by lydiasama** [comment]
Voted! 🚀

**Comment by luongvo** [comment]
@kaungkhantsoe We had integrated deep-link for Compose in a recent project, and I also made a POC here https://github.com/nimblehq/android-templates/pull/531, please check it out. The POC sample shows how to integrate deeplink to the app without from the notification, I think it's good enough to go 👋
